onnxruntime/core/providers/cuda/tensor/gather_elements.cc

```diff
@@ -1,101 +1,214 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#include "gather_elements.h"
-#include "gather_elements_impl.h"
+#include "core/providers/cuda/tensor/gather_elements.h"
+
+#include "core/providers/cuda/tensor/gather_elements_impl.h"
 #include "core/providers/cpu/tensor/utils.h"
 
 namespace onnxruntime {
 namespace cuda {
 
-ONNX_OPERATOR_KERNEL_EX(
-    GatherElements,
-    kOnnxDomain,
-    13,
-    kCudaExecutionProvider,
-    (*KernelDefBuilder::Create())
-        .TypeConstraint("T", DataTypeImpl::AllFixedSizeTensorTypes())
-        .TypeConstraint("Tind", std::vector<MLDataType>{
-                                    DataTypeImpl::GetTensorType<int32_t>(),
-                                    DataTypeImpl::GetTensorType<int64_t>()}),
-    GatherElements);
-
-ONNX_OPERATOR_VERSIONED_KERNEL_EX(
-    GatherElements,
-    kOnnxDomain,
-    11, 12,
-    kCudaExecutionProvider,
-    (*KernelDefBuilder::Create())
-        .TypeConstraint("T", DataTypeImpl::AllFixedSizeTensorTypes())
-        .TypeConstraint("Tind", std::vector<MLDataType>{
-                                    DataTypeImpl::GetTensorType<int32_t>(),
-                                    DataTypeImpl::GetTensorType<int64_t>()}),
-    GatherElements);
+ONNX_OPERATOR_KERNEL_EX(GatherElements, kOnnxDomain, 13, kCudaExecutionProvider,
+                        (*KernelDefBuilder::Create())
+                            .TypeConstraint("T", DataTypeImpl::AllFixedSizeTensorTypes())
+                            .TypeConstraint("Tind", std::vector<MLDataType>{DataTypeImpl::GetTensorType<int32_t>(),
+                                                                            DataTypeImpl::GetTensorType<int64_t>()}),
+                        GatherElements);
+
+ONNX_OPERATOR_VERSIONED_KERNEL_EX(GatherElements, kOnnxDomain, 11, 12, kCudaExecutionProvider,
+                                  (*KernelDefBuilder::Create())
+                                      .TypeConstraint("T", DataTypeImpl::AllFixedSizeTensorTypes())
+                                      .TypeConstraint("Tind",
+                                                      std::vector<MLDataType>{DataTypeImpl::GetTensorType<int32_t>(),
+                                                                              DataTypeImpl::GetTensorType<int64_t>()}),
```

Comment:
this does not looks to be our clang format. 

I open the files locally and it did not format like this, could you please doable check whether you installed other formatting plugin?

Reply:
Can you share the format in your env after you format it?

Reply:
OK, I copied code after your change, the format is compatible with the clang.
I copied code before your change, there won't trigger a re-formatting like this (the diff in this PR):

![image](https://user-images.githubusercontent.com/10530022/165882948-54b429ad-a784-40e8-b385-2551acb4c1d9.png)